dll/win32/shlwapi/string.c

```diff
@@ -2399,8 +2421,13 @@ LPWSTR WINAPI StrFormatByteSizeW(LONGLONG llBytes, LPWSTR lpszDest, UINT cchMax)
     { 100*PB, 107374182.4, 10.00, 1, 'P' }, /* 100 PB */
     { 1000*PB, 1073741824.0, 1.00, 0, 'P' }, /* 1000 PB */
     { 0, 10995116277.76, 100.00, 2, 'E' } /* EB's, catch all */
+#endif
   };
+#ifdef __REACTOS__
+  WCHAR szBuff[64], wszFormat[64];
```

Comment:
You might want to add an API test for the length of this buffer :) By exploiting the function with a large amount of exabytes :trollface:

Reply:
`1 exabytes == 1,000,000,000,000,000,000 bytes` (31 characters).
So, 64 characters is good enough. I will add tests for it.

Reply:
There is already a testcase for `StrFormatByteSizeW`: `modules/rostests/apitests/shlwapi/StrFormatByteSizeW.c` .

Reply:
Adjusted the buffer size to `40`.